README.md

```diff
@@ -351,6 +351,15 @@ nginx_http_template:
           location: /
           proxy_pass: http://backend
       health_check_plus: false
+    proxy_cache_enable: false
```

Comment:
Remember to update the README and `defaults/main.yml` with all the new parameters you're introducing.

Reply:
Done